calculatorBackend/src/ServerStarter/main.cpp

```diff
@@ -0,0 +1,26 @@
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   main.cpp
```

Comment:
Look into removing the exe folder since this contains your new main and cmake to build the executable